views/dataForm.ejs

```diff
@@ -0,0 +1,87 @@
+<%- include("partials/head.ejs") %>
+<%- include("partials/header.ejs") %>
+<%console.log("data", data)%>
+<%
+let eventValue = ""
+let nameValue = ""
+let dateValue = ""
+let descriptionValue = ""
+let id = ""
+if (data !== null) {
+  id = data._id.toString()
+  eventValue = data.event
+  nameValue = data.name
+  descriptionValue = data.description
+   if (data.date !== null) {
+    let day = data.date.getUTCDate()
+    let month = data.date.getUTCMonth()
```

Comment:
You must add 1 to the month because the getUTCMonth method returns the month starting with 0.
Example:
```
let month = data.date.getUTCMonth() + 1
```